models/issues/issue.go

```diff
@@ -146,6 +148,9 @@ type Issue struct {
 
 	// For view issue page.
 	ShowRole RoleDescriptor `xorm:"-"`
+
+	// Time estimate
+	TimeEstimate int64
```

Comment:
I think you need to add a database migration

Reply:
Thank you for pointing that out! Added migrations